lib/managedData/data/countries.json

```diff
@@ -0,0 +1,1496 @@
+[
```

Comment:
I wonder if we could pull this in from a third-party so we don't have to manage it in the future? Though I guess the occasional copy paste wouldn't be a big deal either